src/administrator/components/com_kunena/install/install.script.php

```diff
@@ -17,11 +17,11 @@ class Com_KunenaInstallerScript
 {
 	protected $versions = array(
 		'PHP' => array (
-			'7.0' => '7.0.4',
+			'7.0' => '7.0.3',
 			'5.6' => '5.6.8',
 			'5.5' => '5.5.13',
 			'5.4' => '5.4.13',
-			'0' => '5.4.35' // Preferred version
```

Comment:
That's correct for me, the users needs some time to update their environments, if we didn't notice them they we be mad. Now we can say, we told you some time to update your environment.